projects/plugins/jetpack/modules/videopress/class.videopress-ajax.php

```diff
@@ -41,6 +43,118 @@ public static function init() {
 		return self::$instance;
 	}
 
+	/**
+	 * Ajax method that is used by the VideoPress player to get a token to play a video.
+	 *
+	 * This is used for both logged in and logged out users.
+	 *
+	 * @return void
+	 */
+	public function wp_ajax_videopress_get_playback_jwt() {
+		$guid = filter_input( INPUT_POST, 'guid' );
+		if ( empty( $guid ) ) {
+			wp_send_json_error( array( 'message' => __( 'need a guid', 'jetpack' ) ) );
+			return;
+		}
```

Comment:
Maybe guid should have minimum verification (like the very minimal /^[a-z0-9]+$/i ) and bail like ! is_current_user_authed_for_video (either by adding a common error for both cases or by not returning any errors like the current "is_current_user_authed_for_video" case)

Reply:
done!